packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -396,32 +410,57 @@
                         </oui-tile-description>
                         <oui-action-menu
                             data-compact
-                            data-ng-if="flushCdnState === 'ok' && !hosting.hasCdn && hosting.offer!== 'START_10_M' || hosting.hasCdn && flushCdnState === 'ok' || hosting.hasCdn"
                             data-placement="end"
+                            data-ng-if="flushCdnState === 'ok' && !hosting.hasCdn && hosting.offer!== 'START_10_M' || hosting.hasCdn && flushCdnState === 'ok' || hosting.hasCdn"
                         >
                             <oui-action-menu-item
-                                data-ng-if="flushCdnState === 'ok' && !hosting.hasCdn && hosting.offer!== 'START_10_M'"
+                                data-ng-if="hosting.hasCdn && cdnProperties.version === '2013v1'"
                                 data-href="{{:: $ctrl.$state.href('app.hosting.cdn.order') }}"
-                                ><span
-                                    data-ng-bind=":: (isCdnFree ? 'hosting_dashboard_service_activate_cdn' : 'hosting_dashboard_service_order_cdn') | translate"
-                                ></span
-                            ></oui-action-menu-item>
+                            >
+                                <span
+                                    data-translate="hosting_dashboard_service_change_cdn_version"
+                                ></span>
+                            </oui-action-menu-item>
                             <oui-action-menu-item
-                                data-ng-if="hosting.hasCdn && flushCdnState === 'ok'"
-                                data-on-click="setAction('cdn/flush/hosting-cdn-flush')"
+                                data-ng-if="hosting.hasCdn && cdnProperties.version === 'v2'"
+                                data-href="{{'#/configuration/hosting/'.concat(hosting.serviceName).concat('?tab=MULTISITE')}}"
```

Comment:
Can we inject this instead of hardcoding the url in the view ?

Reply:
resolved